object:
- colour: [255, 0, 0]
  lower_bound: [77, 122, 124]
  max_size: 50000
  min_size: 200
  name: blue
  upper_bound: [117, 212, 243]
- colour: [0, 0, 255]
  lower_bound: [0, 224, 185]
  max_size: 50000
  min_size: 200
  name: hand
  upper_bound: [23, 255, 225]
- colour: [0, 255, 0]
  lower_bound: [145, 133, 188]
  max_size: 50000
  min_size: 200
  name: C2
  upper_bound: [185, 220, 243]
- colour: [0, 255, 255]
  lower_bound: [0, 143, 163]
  max_size: 50000
  min_size: 200
  name: G2
  upper_bound: [72, 231, 255]
